### PR TITLE
Fix buffer size calculation in parseInt.

### PR DESCRIPTION
**Pull request by galpeter**
Fixes issue #279.

JerryScript-DCO-1.0-Signed-off-by: Peter Gal pgal.u-szeged@partner.samsung.com

### CODE REVIEW

File: jerry-core/ecma/builtin-objects/ecma-builtin-global.cpp [hunk 1]
```diff
@@ -110,7 +110,7 @@ ecma_builtin_global_object_parse_int (ecma_value_t this_arg __attr_unused___, /*
   ECMA_TRY_CATCH (string_var, ecma_op_to_string (string), ret_value);
 
   ecma_string_t *number_str_p = ecma_get_string_from_value (string_var);
-  ecma_length_t str_size = ecma_string_get_length (number_str_p);
+  lit_utf8_size_t str_size = ecma_string_get_size (number_str_p);
 
   MEM_DEFINE_LOCAL_ARRAY (utf8_string_buff, str_size + 1, lit_utf8_byte_t);
```
Comment (ruben-ayrapetyan) [inline jerry-core/ecma/builtin-objects/ecma-builtin-global.cpp]: Maybe, we don't need to add the `1` anymore.

Comment (galpeter) [inline jerry-core/ecma/builtin-objects/ecma-builtin-global.cpp]: You are right, we don't need the `+ 1` now. I've removed it and updated the PR.

**Comment by ruben-ayrapetyan** [comment]
Looks good to me

**Comment by LaszloLango** [comment]
LGTM

**Comment by zherczeg** [comment]
LGTM

**Comment by egavrin** [comment]
I'll merge it

**Comment by egavrin** [comment]
Merged at db9b74a1147a3698b3b628bc86fadd4a20372ed5

**Pull request closed without merging**